src/board.js

```diff
@@ -15,15 +15,15 @@ class Board {
     this.checkCombos = false;
   }
 
+  isValidPosition(position) {
```

Comment:
What happens if someone tries to place an x or o on a space that's already taken?

Reply:
Thanks for flagging, we had this in the more complicated version of our start method but didn't include it in the refactored one! We've now added an if...else to handle that possibility.